library/spdm_requester_lib/libspdm_req_handle_error_response.c

```diff
@@ -145,10 +152,25 @@ return_status spdm_handle_response_not_ready(IN spdm_context_t *spdm_context,
 	spdm_context->error_data.token = extend_error_data->token;
 	spdm_context->error_data.rd_tm = extend_error_data->rd_tm;
 
-	return spdm_requester_respond_if_ready(spdm_context, session_id,
-					       response_size, response,
-					       expected_response_code,
-					       expected_response_size);
+	wt = (uint64)pow(2, extend_error_data->rd_exponent)/1000;
+	wt_max = wt * extend_error_data->rd_tm - spdm_context->connection_info.capability.rtt;
+	retry = (wt == 0)? 1 : wt_max / wt;
+	retry = (retry < 1) ? 1 : retry;
+
+	do {
+#ifdef _MSC_VER

```

Comment:
Implementations of system functions belong in os_stub/. This code would call something like libspdm_sleep that would then call into the appropriate system function (Windows, Linux, user-defined, etc). We need to make sure we support different environments, such as RTOSes.